auth/domains.go

```diff
@@ -199,4 +199,7 @@ type DomainsRepository interface {
 
 	// CheckPolicy check policy in domains database.
 	CheckPolicy(ctx context.Context, pc Policy) error
+
+	// DeleteUserPolicies deletes user policies from domains database.
+	DeleteUserPolicies(ctx context.Context, id string) (err error)
```

Comment:
Ranme to `DeletePolicies`.

Reply:
This is different from `DeletePolicies` will delete based on `subject_type`, `subject_id`, `subject_relation`, `object_type` and `object_id` while `DeleteUserPolicies` will delete based on `subject_id` which is `user_id`